source/com.google.android.gms/play-services-safetynet/Additions/Additions.cs

```diff
@@ -29,9 +29,12 @@ public static async Task<bool> ValidateWithGoogle (this SafetyNetApiAttestationR
 
             response.EnsureSuccessStatusCode ();
 
-            var json = JsonObject.Parse (data);
-            if (json.ContainsKey ("isValidSignature"))
-                return json ["isValidSignature"].ToString ().Trim ('"').Equals ("true");
+            JsonDocument doc = JsonDocument.Parse(data);
+            JsonElement root = doc.RootElement;
+            JsonElement signature = root.GetProperty("isValidSignature");
```

Comment:
The original code checks to see if `isValidSignature` exists and returns `false` if it doesn't.  The new code will throw a `KeyNotFoundException` if `isValidSignature` is missing.  I'm not familiar with the intent here, is that acceptable behavior?

Reply:
Fixed. Though I read that `{ “isValidSignature”: true }` is the complete/only response.

https://medium.com/@hargoyal/secure-android-app-with-safetynet-8e367a1c8ad0

https://github.com/googlesamples/android-play-safetynet/blob/master/server/csharp/OnlineVerify.cs#L145-L146

https://stackoverflow.com/questions/39286379/how-to-validate-safety-net-jws-signature-from-header-data-in-android-app